src/component/panels/SummaryPanel/SummaryPanel.tsx

```diff
@@ -261,15 +253,15 @@ function SummaryPanel() {
     );
   }, [correlationsData, handleOnSetMolecularFormula, modal, molecules]);
 
-  const showSetShiftToleranceModal = useCallback(() => {
-    modal.show(
-      <SetShiftToleranceModal
-        onClose={() => modal.close()}
-        onSave={handleOnSetShiftTolerance}
-        previousTolerance={correlationsData.options.tolerance}
-      />,
-    );
-  }, [correlationsData, handleOnSetShiftTolerance, modal]);
+  // const showSetShiftToleranceModal = useCallback(() => {
+  //   modal.show(
+  //     <SetShiftToleranceModal
+  //       onClose={() => modal.close()}
+  //       onSavehandleOnSetShiftTolerance}
+  //       previousTolerance={correlationsData.options.tolerance}
+  //     />,
+  //   );
+  // }, [correlationsDatahandleOnSetShiftTolerance, modal]);
```

Comment:
Don't keep commented code.

Reply:
Sorry this PR should be a draft and not read to be merged, i still refactoring the other Dialog in the summary panels